packages/fuels-types/src/transaction_builders.rs

```diff
@@ -349,7 +349,10 @@ impl CreateTransactionBuilder {
         self
     }
 
-    pub fn set_storage_slots(mut self, storage_slots: Vec<StorageSlot>) -> Self {
+    pub fn set_storage_slots(mut self, mut storage_slots: Vec<StorageSlot>) -> Self {
+        // Storage slots have to be sorted otherwise we'd get a `TransactionCreateStorageSlotOrder`
+        // error.
+        storage_slots.sort();
         self.storage_slots = storage_slots;
         self
     }
```

Comment:
```suggestion
    pub fn set_storage_slots(mut self, storage_slots: Vec<StorageSlot>) -> Self {
        // Storage slots have to be sorted otherwise we'd get a `TransactionCreateStorageSlotOrder`
        // error.
        self.storage_slots = storage_slots.clone().sort();
        self
    }
```

Reply:
Explained in another comment. Resolving